offlineimap/imapserver.py

```diff
@@ -198,7 +198,7 @@ def __md5handler(self, response):
         challenge = response.strip()
         self.ui.debug('imap', '__md5handler: got challenge %s'% challenge)
 
-        passwd = self.__getpassword()
+        passwd = self.__getpassword().encode('ascii')
```

Comment:
It is also reproducible on Python 3.4, while 3.5 works fine.  We could bump version, but I can imagine most people are still having older versions.

I have 3.5 so I did not discover that, but this is definitely a good point.